gsoc/2017/index.md

```diff
@@ -10,18 +10,21 @@ redirect_from: /gsoc/gsoc2017.html
 
 ## Introduction
 
-[Google Summer of Code 2017](https://developers.google.com/open-source/gsoc/) is a program that allows students to contribute to development of open-source projects, mentored by participating organizations.
+[Google Summer of Code](https://developers.google.com/open-source/gsoc/) is a program that allows students to contribute to development of open-source projects, mentored by participating organizations.
 
 Particle physics is an exciting field where large collaborations of scientists collect and analyze the data from high-energy physics experiments, such as the Large Hadron Collider, hosted by the CERN laboratory in Geneva, Switzerland. The questions that we collectively ask are: what are the fundamental blocks that make up our Universe? What is the nature of dark matter and dark energy?  What is the nature of the asymmetry between matter and antimatter? What was early Universe like? To answer these questions, particle physicists build software to simulate and analyze what happens in particle physics detectors.
 
-The CERN software for experiments (CERN-SFT) group has participated in the GSoC since 2011. This year the program expands to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
+The CERN software for experiments (CERN-SFT) group has participated in the GSoC since 2011. In 2017 the program expanded to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
 
-Students interested in participating in the GSoC can find project proposals [here](#Projects).
+Students interested in participating in the GSoC for 2017 were able to chose from various project proposals [here](#Projects).
 
 HSF GSoC Administrators: Sergei Gleyzer <a href="mailto:sergei@cern.ch">sergei@cern.ch</a> and Enric Tejedor Saavedra <a href="mailto:etejedor@cern.ch">etejedor@cern.ch</a>
```

Comment:
We need to add Antoine, who has joined the admin team this year:
- Antoine Perus (perus@lal.in2p3.fr)

Reply:
But that is for the 2017 archive page, which we shouldn't change. Should we put the new coordinators all down on this activities/gsoc.html page maybe?

Reply:
Oh I see, you are right. The new coordinators should then be in the equivalent page for 2018